go/vt/vttablet/endtoend/framework/server.go

```diff
@@ -114,7 +114,7 @@ func StartServer(connParams, connAppDebugParams mysql.ConnParams, dbName string)
 	config.HotRowProtection.Mode = tabletenv.Enable
 	config.TrackSchemaVersions = true
 	config.GracePeriods.ShutdownSeconds = 2
-	config.SchemaReloadIntervalSeconds = tabletenv.Seconds(3)
+	config.SchemaReloadIntervalSeconds = tabletenv.Seconds(2.1)
```

Comment:
Just curious why `2.1` :smile:

Reply:
2 seconds check related to no schema change was added by @vmg to avoid query cache test to fail. To make the test faster made it to `2.1`, i wanted to make it `2.0000001` but that would be too ambitious

Reply:
Then I think `2.1` might also be too ambitious. Might introduce flakyness?